cpp/INIReader.cpp

```diff
@@ -73,7 +73,7 @@ string INIReader::MakeKey(const string& section, const string& name)
 int INIReader::ValueHandler(void* user, const char* section, const char* name,
                             const char* value)
 {
-    INIReader* reader = (INIReader*)user;
+    INIReader *reader = static_cast<INIReader*>(user);
```

Comment:
Thanks -- happy to merge this in. Just one minor nit-pick before I do: can you please change `INIReader *reader ...` back to `INIReader* reader ...` with the space after the `*`? That would be consistent with the rest of the file and it's also more typical C++ style.

Reply:
Fixed! Sorry, my bad...